src/playground/service.clj

```diff
@@ -122,12 +140,12 @@
     ["/login" :get (conj common-interceptors `login-page) :route-name :login]
     ["/login" :post (into common-interceptors [http/json-body (param-spec-interceptor ::session.login/api :form-params) `session.login/perform])]
     ["/logout" :get (conj common-interceptors `logout)]
-    ["/invoices-insert" :get (conj common-interceptors `insert-page)]
+    ["/invoices-insert" :get (into  common-interceptors [authentication-interceptor `insert-page])]
     ["/invoices-insert" :post (into common-interceptors [http/json-body (param-spec-interceptor ::invoices.insert/api :form-params) `invoices.insert/perform])]
```

Comment:
Probably `/invoices-insert" :post ` also needs an authentication-interceptor?

Same for others